[BUG] Project fails to compile when using EE profiler

**Describe the bug**
The PCSX2 project does not compile when the EE Profiler is enabled in [R5900_Profiler.h](https://github.com/PCSX2/pcsx2/blob/0ed46c8b74989011d76389dbf3520bd0f7c195f5/pcsx2/x86/R5900_Profiler.h#L224)

**To Reproduce**
Attempt to compile the project on Windows (x86 or x64)

**Expected behavior**
Project will compile

**GS Settings**
N/A

**Emulation Settings**
N/A

**GS Window Screenshots**
N/A

**System Info (please complete the following information):**
i5-6600k
GTX 1070
Windows 10 64-bit Home
Visual Studio 2019

Compile error:
`1>D:\Git\pcsx2_master\pcsx2\x86\R5900_Profiler.h(373,1): error C2678: binary '*': no operator found which takes a left-hand operand of type 'const x86Emitter::xRegister32' (or there is no acceptable conversion) (compiling source file ..\..\Dump.cpp)
`

Compile log (x64): [compile_log.txt](https://github.com/PCSX2/pcsx2/files/5922688/compile_log.txt)

--- a/pcsx2/x86/R5900_Profiler.h
+++ b/pcsx2/x86/R5900_Profiler.h
@@ -370,7 +370,7 @@ struct eeProfiler {
 		// Compact the 4GB virtual address to a 512KB virtual address
 		if (x86caps.hasBMI2) {
 			xPEXT(ebx, ecx, ptr[&memMask]);
-			xADD(ptr32[(ebx*4) + memStats], 1);
+			xADD(ptr32[(rbx*4) + memStats], 1);
 		}
 	}
 

